twa

```diff
@@ -2,7 +2,7 @@
 
 # twa: a tiny website auditing script
 
-TWA_VERSION="1.5.1"
+TWA_VERSION="1.5.2"
```

Comment:
I prefer separate version commits, so let's not bump this here.

Reply:
reverted